### PR TITLE
RAP-3945

### PR DESCRIPTION
**Pull request by scarabus**

### CODE REVIEW

File: Libs/RaptureAddins/MongoDb/src/main/java/rapture/series/mongo/MongoSeriesStore.java [hunk 1]
```diff
@@ -154,6 +154,7 @@ private void saveDocument(String key, String column, Object val) {
         try {
             @SuppressWarnings("unused")
             Document ret = collection.findOneAndUpdate(dbkey, dbval, options);
+            System.out.println(ret.toJson());
```
Comment (jonathan-mtc) [inline Libs/RaptureAddins/MongoDb/src/main/java/rapture/series/mongo/MongoSeriesStore.java]: Should use logger with debug level

Comment (jonathan-major) [inline Libs/RaptureAddins/MongoDb/src/main/java/rapture/series/mongo/MongoSeriesStore.java]: should use a logging call with debug level

Comment (scarabus) [inline Libs/RaptureAddins/MongoDb/src/main/java/rapture/series/mongo/MongoSeriesStore.java]: That was debug that shouldn't have made it in.

File: Libs/RaptureCore/src/main/java/rapture/repo/file/FileDataStore.java [hunk 2]
```diff
@@ -135,7 +141,8 @@ public void visit(String folderPrefix, RepoVisitor iRepoVisitor) {
     /*
      * (non-Javadoc)
      * 
-     * @see rapture.repo.AbstractKeyStore#matches(java.lang.String, java.lang.String)
+     * @see rapture.repo.AbstractKeyStore#matches(java.lang.String,
```
Comment (jonathan-major) [inline Libs/RaptureCore/src/main/java/rapture/repo/file/FileDataStore.java]: formatting is messed up

**Pull request merged**